Cheza-Dzabala/idove_mobile
# Implement Search UI

### Description
This PR introduces the search User Interface. Search is implemented on a separate page. 

Search uses the `appbar_textfield` plugin.

### Screenshots (if applicable)

#### Search page with the search icon not engaged.
![IMG_1495](https://user-images.githubusercontent.com/13886438/98110376-a7e35f80-1ea7-11eb-9c5c-c9328b1c80c0.PNG)

#### Search page with the search icon engaged.
![IMG_1497](https://user-images.githubusercontent.com/13886438/98111603-84b9af80-1ea9-11eb-9785-730195f8c69b.PNG)


#### Logout button under menu
![IMG_1498](https://user-images.githubusercontent.com/13886438/98111383-2f7d9e00-1ea9-11eb-876e-a8da78b4f209.PNG)

### Type of Change




### Checklist
- [ ] This PR has been tested


- [x] This PR meets all the client standards & conventions

### How To Manually Test

> Clone this repository
> Run `pub get`
> Deploy the application to emulator or physical device. Please see `Readme.md` for instructions on how to do this.
